app/controllers/videos_controller.rb

```diff
@@ -0,0 +1,18 @@
+class  VideosController < ApplicationController
+  #before_action :require_user
+  before_filter :require_user
```

Comment:
`before_action` is the new way :-) you can uncomment the previous line and delete this one.